src/Nethermind/Nethermind.Evm.Test/Tracing/GethLike4byteTracerTests.cs

```diff
@@ -0,0 +1,129 @@
+// SPDX-FileCopyrightText: 2024 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System.Collections.Generic;
+using FluentAssertions;
+using Nethermind.Core.Test.Builders;
+using Nethermind.Evm.Precompiles;
+using Nethermind.Evm.Tracing.GethStyle;
+using Nethermind.Evm.Tracing.GethStyle.Custom.Native.Tracers;
+using NUnit.Framework;
+
+namespace Nethermind.Evm.Test.Tracing;
+
+[TestFixture]
+public class GethLike4byteTracerTests : GethLikeNativeTracerTestsBase
+{
+    [Test]
+    public void Trace_call()
```

Comment:
Can't all of those tests in this class, could be collapsed to one test, with different testcases that would differ in `code` and `expected` Dictionary?

Reply:
Yeah that's a good point, I'll collapse these into one test case

Reply:
It looks like I can't add each case with the `TestCase(...)` notation because `code` and the `expected` dictionary aren't a constant expression, 'typeof()' expression or array creation expression of an attribute parameter type. Do you have any other ideas on simplifying these tests?

Reply:
you can do it with `TestCaseSource`